tank_battalion/player.pde

```diff
@@ -7,7 +7,7 @@ class Player {
   int move_speed = 4;
   int collider_id;
   float cooldown = FIRE_COOLDOWN;
-  String direction = "up";
+  int direction = 1;
```

Comment:
I understand that you don't regard string as a suitable type here, but I wonder why you've chosen int instead. I think the string variant was a bit more clear in terms of what value means what. Wouldn't it be better to use an enum type?

Reply:
I changed it because the enemies already used a system with integers (`direction = random(1,5)`). It would be easier to change the datatype in `player` then to use random strings.